src/applications/personalization/profile/components/ProfileInfoTable.jsx

```diff
@@ -30,6 +31,7 @@ const ProfileInfoTable = ({
     'margin--0',
     'padding-x--2',
     'padding-y--1p5',
+    'vads-u-font-size--h3',
```

Comment:
Whoops, this no longer needs the `vads-u` prefix

Reply:
no problem will adjust that